src/Faithlife.Tracing.Data/TracingDbConnection.cs

```diff
@@ -11,9 +12,9 @@ namespace Faithlife.Tracing.Data
 {
 	public class TracingDbConnection : DbConnection
 	{
-		public static DbConnection Create(DbConnection connection, ITraceProvider traceProvider) => new TracingDbConnection(connection, traceProvider);
+		public static DbConnection Create(DbConnection connection, ITraceProvider traceProvider) => new TracingDbConnection(connection ?? throw new ArgumentNullException(nameof(connection)), traceProvider);
```

Comment:
`DataTracing.CreateConnection`? `DbTracing.CreateConnection`? No need to leak the `TracingDbConnection` class, unless it has new members the client might need.